.circleci/config.yml

```diff
@@ -121,15 +134,62 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+      - unit_test:
+          requires:
+            - build
+      - deploy_canary:
+          filters:
+            branches:
+              only: develop
+          requires:
+            - unit_test
+      - deploy_branch:
+          filters:
+            branches:
+              ignore:
+                - master
+                - develop
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+          requires:
+            - unit_test
+  build_and_deploy_i18n:
```

Comment:
would it make sense to split this workflow into one for release tags + master, and one for support, feature-i18n, and release?

Reply:
Ya, we could do that

Reply:
that might simplify some of the filtering logic